system/include/emscripten/threading.h

```diff
@@ -148,6 +148,15 @@ void emscripten_conditional_set_current_thread_status(EM_THREAD_STATUS expectedS
 // When thread profiler is not enabled (not building with --threadprofiling), this is a no-op.
 void emscripten_set_thread_name(pthread_t threadId, const char *name);
 
+// Gets the stored pointer to a string representing the canvases to transfer to the created thread.
+int emscripten_pthread_attr_gettransferedcanvases(const pthread_attr_t *a, const char **str);
+
+// Specifies a comma-delimited list of canvas DOM element IDs to transfer to the thread to be created.
+// Note: this pointer is weakly stored (not copied) to the given pthread_attr_t, so must be held alive until
+// pthread_create() has been called. If 0 or "", no canvases are transferred. The special value "#canvas" denotes
+// the element stored in Module.canvas.
+int emscripten_pthread_attr_settransferedcanvases(pthread_attr_t *a, const char *str);
```

Comment:
Typo: 'transferred', not 'transfered'. At least in API functions we shall not typo.